src/debug.c

```diff
@@ -3530,6 +3838,140 @@ static void DebugAction_Give_DayCareEgg(u8 taskId)
         TriggerPendingDaycareEgg();
 }
 
+// *******************************
+// Actions TimeMenu
+
+static void DebugAction_TimeMenu_Morning(u8 taskId)
+{
+    DebugAction_DestroyExtraWindow(taskId);
+    u32 MorningBegin = MORNING_HOUR_BEGIN;
+
+    FakeRtc_ForwardTimeTo(MorningBegin, 0, 0);
+    Debug_DestroyMenu_Full(taskId);
+    SetMainCallback2(CB2_LoadMap);
+}
+
+static void DebugAction_TimeMenu_Day(u8 taskId)
+{
+    DebugAction_DestroyExtraWindow(taskId);
+    u32 DayBegin = DAY_HOUR_BEGIN;
+    
+    FakeRtc_ForwardTimeTo(DayBegin, 0, 0);
+    Debug_DestroyMenu_Full(taskId);
+    SetMainCallback2(CB2_LoadMap);
+}
+
+static void DebugAction_TimeMenu_Evening(u8 taskId)
+{
+    DebugAction_DestroyExtraWindow(taskId);
+    u32 EveningBegin = EVENING_HOUR_BEGIN;
+    
+    FakeRtc_ForwardTimeTo(EveningBegin, 0, 0);
+    Debug_DestroyMenu_Full(taskId);
+    SetMainCallback2(CB2_LoadMap);
+}
+
+static void DebugAction_TimeMenu_Night(u8 taskId)
+{
+    DebugAction_DestroyExtraWindow(taskId);
+    u32 NightBegin = NIGHT_HOUR_BEGIN;
+    
+    FakeRtc_ForwardTimeTo(NightBegin, 0, 0);
+    Debug_DestroyMenu_Full(taskId);
+    SetMainCallback2(CB2_LoadMap);
+}
+
+static void DebugAction_TimeMenu_Sunday(u8 taskId)
+{
+    DebugAction_DestroyExtraWindow(taskId);
+    struct SiiRtcInfo *rtc = FakeRtc_GetCurrentTime();
+    
+    u32 weekdayCurrent = rtc->dayOfWeek;
+    u32 daysToAdd;
+    daysToAdd = ((0 - weekdayCurrent) + 7) % 7;
```

Comment:
```suggestion
    u32 daysToAdd = ((WEEKDAY_SUN - rtc->dayOfWeek) + WEEKDAY_COUNT) % WEEKDAY_COUNT;
```

And analogously for the other functions for the other days of the week